cloudbuild-pr.yaml

```diff
@@ -97,7 +97,7 @@ steps:
       - run
       - '--network=cloudbuild'
       - test_docker
-      - '--enable_caching=0'
+      - '--enable_caching=1'
```

Comment:
Was this intended? We disable caching to make sure changes to containers are tested.

Reply:
This was intentional to speed up build times on repeated builds.

My expectation is that Vertex takes the SHA of an image into account when deciding to use a cached step. If this isn't the case (or is uncertain) I will undo the change.

Reply:
Unfortunately Vertex doesn't take the SHA and only uses image name for caching :(

Reply:
Reverted.